setup.py

```diff
@@ -14,8 +14,9 @@
     'pillow',
     'django-filer',
     'django-sizefield',
-    'Django>=1.8,<2.0',
     'openpyxl<=2.4.9',  # 2.5.0b1 is raising "ImportError: cannot import name '__version__'"
+    'Django>=1.8,<2.0',
+    'django-polymorphic>=1.3,<2.0',  # only supports Django>=1.8,<2.0
```

Comment:
can't limit this, some apps run with polymorphic 2.0 which still supports django 1.x

Reply:
yeah, django-polymorphic 2.0 supports django>=1.11. But with aldryn-forms we want to support Django>=1.8,<2.0. I can do separate test requirements to fix failing tests because of django-polymorphic version, but then we allow users to install aldryn-forms on django18 with django-polymorphic 2.0, and it will not work for them. Are you ok with that? Or is there a solution for that?

Reply:
polymorphic is not a requirement of this app.
so for me is ok to leave it out.